packages/fluentui/docs/src/components/ComponentDoc/ComponentDocSee.tsx

```diff
@@ -15,12 +15,13 @@ const ComponentDocSee: any = ({ displayName }) => {
   return (
     <List styles={listStyle}>
       {/* Heads up! Still render empty lists to reserve the whitespace */}
-      <List.Item>
+      <List.Item index={0}>
```

Comment:
`index` is a required prop on `ListItem` for Children API, https://github.com/microsoft/fluent-ui-react/pull/2207

Previous typings were not correct because we exported `ListItemProps` only:
https://github.com/microsoft/fluentui/blob/c46010712f3515f5f6fba9966b5c18546ae94cea/packages/fluentui/react-northstar/src/components/List/ListItem.tsx#L92
https://github.com/microsoft/fluentui/blob/c46010712f3515f5f6fba9966b5c18546ae94cea/packages/fluentui/react-northstar/src/components/List/ListItem.tsx#L287